commonItems/GameVersionFactory.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace commonItems {
+    public class GameVersionFactory : Parser {
```

Comment:
This seems redundant with the constructor that takes a BufferedReader. It exists in the C++ commons because some of the converters disagree on which way is preferable. Pick which one you think you'll use and delete the other.

Reply:
I like it! Factories might still be useful for any classes that get created a lot, but for something like GameVersion this would be my preferred style.